features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APICreateMenu.jsx

```diff
@@ -5,40 +5,68 @@ import MenuList from '@material-ui/core/MenuList';
+                        <ListItemText
+                            primary={<FormattedMessage
+                                id='design.a.new.rest.api'
+                                defaultMessage='Design a New REST API'
+                            />}
+                            secondary={<FormattedMessage
+                                id='design.and.prototype.a.new.rest.api'
+                                defaultMessage='Design and prototype a new REST API'
+                            />}
+                        />
                     </Link>
                 </ListItem>
                 <Divider />
                 <ListItem>
                     <Link to='/apis/create/swagger'>
                         <ListItemText
-                            primary='I Have an Existing REST API'
-                            secondary='Use an existing REST endpoint or Swagger definition'
+                            primary={<FormattedMessage
+                                id='i.have.an.existing.api'
+                                defaultMessage='I Have an Existing REST API'
+                            />}
+                            secondary={<FormattedMessage
+                                id='use.an.existing.rest.endpoint.or.Swagger.definition'
+                                defaultMessage='Use an existing REST endpoint or Swagger definition'
+                            />}
                         />
                     </Link>
                 </ListItem>
                 <Divider />
                 <ListItem>
                     <Link to='/apis/create/wsdl'>
                         <ListItemText
-                            primary='I Have a SOAP Endpoint'
-                            secondary='Use an existing SOAP or Import the WSDL'
+                            primary={<FormattedMessage
+                                id='i.have.a.soap.endpoint'
+                                defaultMessage='I Have a SOAP Endpoint'
+                            />}
+                            secondary={<FormattedMessage
+                                id='use.an.existing.soap.or.import.the.wsdl'
+                                defaultMessage='Use an existing SOAP or Import the WSDL'
+                            />}
                         />
                     </Link>
                 </ListItem>
                 <Divider />
                 <ListItem>
-                    <Link to='/apis/create/rest'>
+                    <Link to='/apis/create/websocket'>
                         <ListItemText
-                            primary='Design New Websocket API'
-                            secondary='Design and prototype a new WebSocket API'
+                            primary={<FormattedMessage
+                                id='design.new.websocket.api'
+                                defaultMessage='Design New WebSocket API'
```

Comment:
Word New should start with a simple letter.

Reply:
From UX point of view it is meant to be camel case.